acme/acme/messages.py

```diff
@@ -315,7 +315,7 @@ class RegistrationResource(ResourceWithURI):
     """Registration Resource.
 
     :ivar acme.messages.Registration body:
-    :ivar unicode new_authzr_uri: URI found in the 'next' ``Link`` header
+    :ivar unicode new_authzr_uri: Deprecated. Do not use.
```

Comment:
I don't think this matters, however, leaving the `new_*_uri` fields unchanged means they're still included in the serialized JSON. For instance:
```
{'body': {'key': ..., 'new_authzr_uri': None, 'terms_of_service': ...
```
I'm not sure if this is a problem since I don't believe we ever send the full registration resource or authorization resource to the ACME server, but perhaps it should be fixed anyway. This can be done by setting `default=None` and `omitempty=True` when defining the field as described at https://github.com/certbot/certbot/blob/master/acme/acme/jose/json_util.py#L41.

I'll defer to your judgement of whether or not you think we should do this.